docs/projects/cn0506/index.rst

```diff
@@ -0,0 +1,396 @@
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Below are the Programmable Logic interrupts used in this project.
+
+=========================== === ============ =============
+Instance name               HDL Linux Zynq   Actual Zynq
+=========================== === ============ =============
+axi_hdmi_dma/irq*           15  59           91 
+axi_iic_main/iic2intc_irpt* 14  58           90
+axi_iic_fmc/iic2intc_irpt*  11  55           87
+=========================== === ============ =============
+
+.. admonition:: Legend
+   :class: note
+
+   -  ``*``  instantiated only for Zed and ZC706 carriers
+   -  ``**`` instantiated only for Zed carrier
+
+Building the HDL project
+-------------------------------------------------------------------------------
+
+The design is built upon ADI's generic HDL reference design framework.
+ADI distributes the bit/elf files of these projects as part of the
+:dokuwiki:`ADI Kuiper Linux <resources/tools-software/linux-software/kuiper-linux>`.
+If you want to build the sources, ADI makes them available on the
+:git-hdl:`HDL repository </>`. To get the source you must
+`clone <https://git-scm.com/book/en/v2/Git-Basics-Getting-a-Git-Repository>`__
+the HDL repository, and then build the project as follows:.
+
+**Linux/Cygwin/WSL**
+
+.. code-block::
+   :linenos:
+
+   user@analog:~$ cd hdl/projects/cn0506/zed
+   user@analog:~/hdl/projects/cn0506/zed$ make INTF_CFG=MII
+
+The result of the build, if parameters were used, will be in a folder named
+by the configuration used:
+
+if the following command was run
+
+``make INTF_CFG=MII``
+
+then the folder name will be:
+
+``INTF_CFGMII``
+
+A more comprehensive build guide can be found in the :ref:`build_hdl` 
+user guide.
+
+Resources
+-------------------------------------------------------------------------------
+
+Hardware related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  Product datasheet: :adi:`CN0506`
+-  ADIN1300 datasheet: :adi:`ADIN1300` 
+
+HDL related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  :git-hdl:`CN0506 HDL project source code <projects/cn0506>`
+
+.. list-table::
+   :widths: 30 35 35
+   :header-rows: 1
+
+   * - IP name
+     - Source code link
+     - Documentation link
+   * - AXI_CLKGEN *
+     - :git-hdl:`library/axi_clkgen`
+     - :ref:`here <axi_clkgen>`
+   * - AXI_HDMI_TX *
+     - :git-hdl:`library/axi_hdmi_tx`
+     - :ref:`here <axi_hdmi_tx>`
+   * - AXI_I2S_ADI **
+     - :git-hdl:`library/axi_i2s_adi`
+     - ---
+   * - AXI_SYSID
+     - :git-hdl:`library/axi_sysid <library/axi_sysid>`
+     - :ref:`here <axi_sysid>`
+   * - SYSID_ROM
+     - :git-hdl:`library/sysid_rom <library/sysid_rom>`
+     - :ref:`here <axi_sysid>`
+   * - AXI_SDDIF_TX *
+     - :git-hdl:`library/axi_spdif_tx`
+     - ---
+   * - UTIL_I2C_MIXER **
+     - :git-hdl:`library/util_i2c_mixer`
+     - ---
+   * - UTIL_MII_TO_RMII ***
```

Comment:
`**` is the syntax of emphasis, you should escape when using as a plain triple asterisk, to avoid warnings
```suggestion
   * - UTIL_MII_TO_RMII \***
```